### DISCUSSION TITLE
Keycloak identity provider, how to forward errors to client?

### DISCUSSION THREAD

**Discussion by FelipeEmerim** [keycloak/keycloak#9870] [Q&A]
Hi,

We are using keycloak with the Azure AD identity provider. We hide keycloak's login page by disabling forms in the browser flow and configuring an identity provider redirector as required. However, if the user denies consent on Azure AD, he will be redirected to the keycloak error page. Is there a way to make keycloak redirect the error to the client so that the client can display the appropriate page?

**Comment by Gabriel01-jpg** [comment]
Ward if someone have the answer.
